### ISSUE TITLE
"Log in via Github to add your work" broken.

### ISSUE THREAD

**Issue by Potherca** [stillmaintained/stillmaintained#35]
When I click the "login with github" button on the frontpage I get an error on the page I: 
"Woah, something went wrong
We’ve been notified of the issue, but you can always report additional information. Do you want to go back to the homepage?"

The url is I am at when the error occurs is `http://stillmaintained.com/auth/github/callback?code=5aa65c5f11ddf20f7a0d&state=d8d8e6d51a0f61229080367b994cf58a6379bb0378097e4a`

I'd like to login so I thought I'd better report this, although I don't know what additional information I could give you...

**Comment by jadb** [comment]
Hi @potherca, 

I just tried it and it's working fine for me. Can you please try again? Maybe try with a different browser?

Thanks,

Jad

**Comment by doriath** [comment]
Hi,

Sorry for late response. I could not reproduce this issue either (I tried for new and already existing users). Could you confirm that the problem still exists?

While trying to debug this problem, I found out two problems in current system:
1. On every restart we lose all our logs (we are truncating the old log file instead of appending to it).
2. I will create the airbrake account and configure it on production, to make sure that we are notified about such problems automatically and we get the full stack trace of the issue.

Thanks for reporting the issue!
Tomasz

**Comment by jadb** [comment]
Thanks @doriath as usual! :+1:

**Comment by Potherca** [comment]
the problem occurs for me regardless of the browser I use. I've tried this with Firefox 11, Chromium 18 and Chrome 21. I am on LinuxMint 10 (although I suspect that will not proof to be relevant).

I've tried with and without already being logged in to Github, all lead to the Woah/Error.

Under my Github application settings, StillMaintained shows up in the Authorized applications list, so access is granted... it just doesn't seem to return correctly.
